package/MDAnalysis/core/AtomGroup.py

```diff
@@ -3799,16 +3871,19 @@ def add(self, other):
             def update(self, other):
                 self.ats.update(other.ats)
 
-        f = dict.fromkeys(self.atoms, None)  # each atom starts with its own list
+        f = dict.fromkeys(self.atoms,
+                          None)  # each atom starts with its own list
 
         for a1, a2 in bonds:  # Iterate through all bonds
             if not (f[a1] or f[a2]):  # New set made here
                 new = _fragset([a1, a2])
                 f[a1] = f[a2] = new
-            elif f[a1] and not f[a2]:  # If a2 isn't in a fragment, add it to a1's
+            elif f[a1] and not
+            f[a2]:  # If a2 isn't in a fragment, add it to a1's
```

Comment:
Ah I noticed that `yapf` doesn't like these trailing comments. This is also the reason why some of the formatting comes out weird.